packages/vscode-extension/src/extension.ts

```diff
@@ -156,17 +154,13 @@ function registerActivateCommands(context: vscode.ExtensionContext) {
     "createProject"
   );
   context.subscriptions.push(
-    vscode.commands.registerCommand("fx-extension.getNewProjectPath", async (...args) => {
-      if (!isSupportAutoOpenAPI()) {
```

Comment:
Won't we support old version vscode that doesn't have autoOpen API?

Reply:
I've updated the required VS Code version in `package.json`. It should only affect a very small portion of users since VS Code is auto upgraded by default.